packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWTabs/CWTab.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import clsx from 'clsx';
+
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWTag } from 'views/components/component_kit/new_designs/cw_tag';
+
+import './CWTab.scss';
+
+interface CWTabProps {
+  label: string;
+  isSelected: boolean;
+  onClick: () => void;
+  isDisabled?: boolean;
+  showTag?: boolean;
+}
+
+const CWTab = ({
+  label,
+  isDisabled,
+  isSelected,
+  onClick,
+  showTag,
+}: CWTabProps) => {
+  return (
+    <button
+      className={clsx('CWTab', { isSelected, isDisabled })}
```

Comment:
just confirming if we want to use `clsx` or `getClasses`?

Reply:
I see that for design system we use `getClasses` so I am gonna change this 👍